tests/bfd/test_bfd.py

```diff
@@ -481,3 +490,35 @@ def test_bfd_multihop(request, rand_selected_dut, ptfhost, tbinfo,
         duthost.shell(cmd_buffer, module_ignore_errors=True)
         ptfhost.command('supervisorctl stop bfd_responder')
         ptfhost.file(path=BFD_RESPONDER_SCRIPT_DEST_PATH, state="absent")
+
+
+@pytest.mark.parametrize('ipv6', [False, True], ids=['ipv4', 'ipv6'])
+def test_bfd_echo_mode(request, rand_selected_dut, ptfhost, tbinfo, ipv6):
```

Comment:
Can you please point to me where the actual BFD session is being created.
From what  i understand you are simply setting vtysh -c 'configure terminal' -c 'bfd' -c 'bfd peer {neighbor IP}' -c 'echo-mode'
and then you do 
vtysh -c 'show bfd peer'"
Is the intent of the test simply to test i f the echo mode is configurable.

Reply:
This command establishes the BFD neighborship first and then enables the echo-mode on the peer. The breakdown of the command is as follows:
-c bfd (to instantiate the bfd instance)
-c bfd peer (neighbor IP)
These two commands enable the bfd session.

-c echo-mode (enabling echo-mode on the peer)
This command is to configure the peer.

Refer to lines 326-328 in bfd_test.py.